Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+            if (!HasScopedAttributesEmpty(attributesJson))
+            {
+                return new Dictionary<string, string>();
+            }
+            var result = new Dictionary<string, string>();
+            var attributes = JsonUtility.FromJson<ScopedAttributesContainer>(attributesJson);
+            foreach (var attributeKey in attributes.Keys)
+            {
+                var value = PlayerPrefs.GetString(string.Format(ScopedAttributesPattern, attributeKey), string.Empty);
+                result[attributeKey] = value;
+            }
+
+            // extend scoped attributes with legacy attributes stored by Backtrace-Unity library in previous versions
+            IDictionary<string, string> legacyAttributes = new Dictionary<string, string>()
+                    {
+                        { MachineUuidKey, "guid" },
+                        { VersionKey, "application.version" },
+                        { SessionKey, BacktraceMetrics.ApplicationSessionKey }
+                    };
+
+            foreach (var legacyAttribute in legacyAttributes)
+            {
+                string legacyAttributeValue = PlayerPrefs.GetString(legacyAttribute.Key, string.Empty);
+                if (!string.IsNullOrEmpty(legacyAttributeValue))
+                {
+                    PlayerPrefs.DeleteKey(legacyAttribute.Key);
+                    result[legacyAttribute.Value] = legacyAttributeValue;
+                }
+            }
+            return result;
+        }
+
+        /// <summary>
+        /// Adds attributes to scoped registry and to native clietn
+        /// </summary>
+        /// <param name="key">attribute key</param>
+        /// <param name="value">attribute value</param>
+        private void AddAttributes(string key, string value)
+        {
+            if (_captureNativeCrashes)
+            {
+                AddNativeAttribute(key, value);
+            }
+            AddScopedAttribute(key, value);
+        }
+
+        /// <summary>
+        /// Adds dictionary of attributes to player prefs for windows crashes captured by unity crash handler
+        /// </summary>
+        /// <param name="atributes">Attributes</param>
+        internal void AddScopedAttributes(IDictionary<string, string> attributes)
+        {
+            if (!_configuration.SendUnhandledGameCrashesOnGameStartup)
+            {
+                return;
+            }
+            var attributesContainer = new ScopedAttributesContainer();
+            foreach (var attribute in attributes)
+            {
+                attributesContainer.Keys.Add(attribute.Key);
+                PlayerPrefs.SetString(string.Format(ScopedAttributesPattern, attribute.Key), attribute.Value);
+            }
+            PlayerPrefs.SetString(ScopedAttributeListKey, JsonUtility.ToJson(attributesContainer));
+        }
+
+        /// <summary>
+        /// Adds attribute to player prefs for windows crashes captured by Unity crash handler
```

Comment:
crashpad too?

Reply:
Only Unity crash handler - Crashpad attributes are stored by different flow and inside the native client.